### PR TITLE
Add page for PR wranglers

### PR DESCRIPTION
**Pull request by celestehorgan**
Closes https://github.com/kubernetes/website/issues/22276
Umbrella issue: https://github.com/kubernetes/website/issues/21687

This PR creates a new page for PR wrangler information. It also corrects some links and adds a what's next section to the Roles and Responsibilities page.

- [Old topic](https://kubernetes.io/docs/contribute/advanced/#be-the-pr-wrangler-for-a-week)
- New topic

### CODE REVIEW

**Comment by tengqm** [comment]
Where are we heading with these PRs?
I have just synchronized the English contribution pages to Chinese. (#22241, #22242, #22243, #22245, #22246).
It took me 26 hours to retranslate all the contents. They have to be retranslated because git is not smart enough to tell me which line has been changed, moved (from where to where) ...
Any compelling reasons we keep reorganizing these guides?

File: content/en/docs/contribute/participate/pr-wranglers.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+- The author has not responded to comments or feedback in 2 or more weeks.
+
+Don't be afraid to close pull requests. Contributors can easily reopen and resume works in progress. Often a closure notice is what spurs an author to resume and finish their contribution.
+
+To close a pull request, leave a `/close` comment on the PR.
+
+{{< note >}}
+
+The [`fejta-bot`](https://github.com/fejta-bot) bot marks issues as stale after 90 days of inactivity. After 30 more days it marks issues as rotten and closes them.  PR wranglers should close issues after 14-30 days of inactivity.
```
Comment (sftim) [inline content/en/docs/contribute/participate/pr-wranglers.md:67]: > PR wranglers should close issues after 14-30 days of inactivity.

Really? That's not what I'd do. I'm mostly happy to have @fejta or their robot assistant mark issues as stale and have these expire.

There are some long-standing, triaged issues that I'd like to keep open even if these don't get addressed in the next 30 days. For example, there was for a long time a feature request to add a concept page for ConfigMap. That particular issue had lifecycle=frozen, so maybe we need to be more active in freezing important issues?

File: content/en/docs/contribute/participate/roles-and-responsibilties.md [hunk 2]
```diff
@@ -171,7 +171,7 @@ Approvers and SIG Docs leads are the only ones who can merge pull requests into
 - Visit the Netlify page preview for a PR to make sure things look good before approving.
 
 - Participate in the [PR Wrangler rotation schedule](https://github.com/kubernetes/website/wiki/PR-Wranglers) for weekly rotations. SIG Docs expects all approvers to participate in this
-rotation. See [Be the PR Wrangler for a week](/docs/contribute/advanced#be-the-pr-wrangler-for-a-week)
+rotation. See [PR wranglers](/docs/contribute/participating/pr-wranglers)
```
Comment (sftim) [inline content/en/docs/contribute/participate/roles-and-responsibilties.md:174]:
```suggestion
rotation. See [PR wranglers](/docs/contribute/participating/pr-wranglers/).
```

File: content/en/docs/contribute/participate/roles-and-responsibilties.md [hunk 3]
```diff
@@ -188,4 +188,10 @@ To apply:
 
 2. Assign the PR to one or more current SIG Docs approvers.
 
-If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
\ No newline at end of file
+If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
```
Comment (sftim) [inline content/en/docs/contribute/participate/roles-and-responsibilties.md:191]:
```suggestion
If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [@k8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
```
Could also link to https://github.com/k8s-ci-robot (which then links to https://github.com/kubernetes/test-infra/tree/master/prow#bots-home)

File: content/en/docs/contribute/participate/roles-and-responsibilties.md [hunk 4]
```diff
@@ -188,4 +188,10 @@ To apply:
...
+
+## {{% heading "whatsnext" %}}
+
+Learn about specialized roles in SIG Docs:
```
Comment (sftim) [inline content/en/docs/contribute/participate/roles-and-responsibilties.md:195]: Possibly:
```suggestion
Learn about specialized roles in SIG-Docs:
```

File: content/en/docs/contribute/participate/roles-and-responsibilties.md [hunk 5]
```diff
@@ -188,4 +188,10 @@ To apply:
...
+
+- [PR wranglers](/docs/contribute/participating/pr-wranglers)
```
Comment (sftim) [inline content/en/docs/contribute/participate/roles-and-responsibilties.md:197]: The _role_ is approver, I think. All approvers become PR wranglers on a rota.

**Comment by sftim** [comment]
Responding to https://github.com/kubernetes/website/pull/22284#issuecomment-653285453

I'd keen to see the right content land @tengqm. Although revisions trigger work for localization teams, I'm not at all keen to keep pages that are outdated for the reason that they are also outdated in several other languages.

File: content/en/docs/contribute/participate/roles-and-responsibilties.md (hunk 4 above)
Comment (tengqm) [reply content/en/docs/contribute/participate/roles-and-responsibilties.md:195]: Nope. We are sticking to 'SIG Docs'.
https://kubernetes.io/docs/contribute/style/style-guide/#kubernetes-io-word-list

**Comment by tengqm** [comment]
@sftim I am all for content improvement for the docs or website. What I'm unhappy about this is that we are splitting pages, moving pages, while at the same time, editing contents. The git tool is stupid enough to mess up all things when doing a diff. When syncing English content changes to any localization, we lose track of all previous translations. For English content, it looks pretty easy and trivial change to shuffle contents. For localization teams, this is a nightmare.

File: content/en/docs/contribute/participate/pr-wranglers.md (hunk 1 above)
Comment (celestehorgan) [reply content/en/docs/contribute/participate/pr-wranglers.md:67]: This was the original text – I have no comment personally. 

If you had to summarize your 'stance' on when/how we should close issues, how would you do it?

Comment (sftim) [reply content/en/docs/contribute/participate/pr-wranglers.md:67]: Ah, if it's existing text then I'm fine to leave it! Another PR can adjust the wording.

**Comment by celestehorgan** [comment]
@tengqm 

Also responding to https://github.com/kubernetes/website/pull/22284#issuecomment-653538158 

These PRs are really finishing up the work that I started in previous PRs – namely, reorganizing this entire section and making sure the content is up to date. When I started this work, I was concerned about downstream translations for the reasons that you stated. I knew it would take multiple PRs/reorganizations to get this right and complete, after all. I was told not to worry about that :/ 

That said: I (personally) think there's more content re-organization to do in the k8s docs, so maybe we need to think about how to solve this.  Maybe a frontmatter flag or `<!--comment-->` in the .md of a page, i.e. `<!-- translation hold -->` on pages undergoing re-organization, that gets removed when the author deems that work on a section is 'done'? The translation sync script could look for that flag and ignore changes in those files until then?

**Comment by sftim** [comment]
We have a comment shortcode that might be useful for warning localization teams about details such as likely future changes.

File: content/en/docs/contribute/participate/roles-and-responsibilties.md (hunk 5 above)
Comment (celestehorgan) [reply content/en/docs/contribute/participate/roles-and-responsibilties.md]: @sftim addressed this in the latest commit.

**Pull request merged**